modules/apps/feature-flag/feature-flag-web/src/FeatureFlags.testcase

```diff
@@ -0,0 +1,79 @@
+@component-name = "feature flags"
+definition {
+
+	property portal.acceptance = "true";
```

Comment:
Remember to add property custom.properties:

feature.flag.LPS-xxxxxxx=false
feature.flag.LPS-xxxxxx.status=beta
feature.flag.LPS-xxxxxx.title=Terms and Conditions pages
feature.flag.LPS-xxxxxx.description=Create your own terms and conditions pages

Reply:
feature.flag.LPS-yyyyyy=false
feature.flag.LPS-yyyyyy.status=beta
feature.flag.LPS-yyyyyy.dependencies=LPS-xxxxxx